spec/requests/posts_spec.rb

```diff
@@ -3,7 +3,21 @@
 RSpec.describe 'Posts', type: :request do
   describe 'GET /index' do
     it 'returns http success' do
-      get '/posts/index'
+      get '/posts'
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'GET users/:id/post/:id' do
+    it 'returns http success' do
+      get '/users/1/posts/7'
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'GET users/:id/post' do
+    it 'returns http success' do
+      get '/users/1/posts'
       expect(response).to have_http_status(:success)
     end
   end
```

Comment:
- [ ] Kindly organize your tests here. You should only have two `describe` groupings for post index and show pages.

  ```
  describe 'GET #index' do
    it '...' do
      ...
    end
    ....
  end

  describe 'GET #show' do
    it '...' do
      ...
    end
    ...
  end
  ```

- [ ] Besides checking for the status you should also check for text on the respective pages and ensure the correct template is being rendered:

  ```
  expect(response).to render_template(:show)
  ```
  ```
  expect(response.body).to include("Welcome to user index page")
  ```

Reply:
Tests for all the html.erb files have bee created and out in a folder `view`